bulbs/super_features/api.py

```diff
@@ -1,11 +1,23 @@
 from django.conf.urls import url
 
-from bulbs.super_features.viewsets import RelationViewSet
+from rest_framework import routers
+
+from bulbs.super_features.views import RelationViewSet
+from bulbs.super_features.viewsets import SuperFeatureViewSet
+
+api_v1_router = routers.DefaultRouter()
+api_v1_router.register(
+    r"super-feature",
```

Comment:
Should standardize on naming for both routes + code. Is it one word "superfeature" or two words "super_feature/super-feature/SuperFeature"